### ISSUE TITLE
Prepare quote (and buy) for Tokens

### ISSUE THREAD

**Issue by PeterJCLaw** [srobo/competition-team-minutes#179]
Aim to have them for the London Tech Day if we can.

We'll store them until the competition.

**Comment by RealOrangeOne** [comment]
This feels like a task better suited in https://github.com/srobo/tasks, thoughts?

**Comment by PeterJCLaw** [comment]
Yeah, possibly. Ideally we'd import all the competition tasks at some point, which likely includes doing this too.

**Comment by PeterJCLaw** [comment]
Realistically I'm not currently working on this.
